### DISCUSSION TITLE
theme preview side-by-side

### DISCUSSION THREAD

**Discussion by Dunky13** [ghostty-org/ghostty#3895] [Feature Requests, Ideas] [answered]
Let's say I want to find a new there, I can use the theme preview to see how each theme looks. But when I scroll further than 10 more themes, I've forgotten how my previously liked theme looked like.
For example, I liked `Bright Lights` but then I came across `Chalk` and was like, this seems nice too. But how is it different again from Bright Lights?

What would be nice to have, is you can pin Either the "left" or the "right" theme (or top/bottom, or whatever) And when you go down the list, the non-pinned theme changes, thus you can easily compare. If you like the new one better, you switch the pin, and continue down the list.

By the time you reach the end, you have found the theme you like.

Hope it makes sense.

**Comment by mitchellh** [comment, marked as answer]
I'd like to propose you open a second window for this.

**Reply by Dunky13** [reply]
Fair enough 😅
